apps/app/src/views/routes/Order.tsx

```diff
@@ -339,354 +373,446 @@ export const Order = () => {
     );
   }
 
+  const fills = order ? uniqueFills(order) : [];
+
   return (
-    <Page header="Order">
-      <Card>
-        <CardHeader>
-          <Stack direction={{ base: 'column', md: 'row' }} justify="space-between" width="full">
+    <>
+      <TicketModal
+        isOpen={isTicketModalOpen}
+        onClose={() => setIsTicketModalOpen(false)}
+        subject={`Issue with order for ${order?.patient?.name?.full}`}
+        body={
+          <>
+            <Text>Start an email thread with the Photon team to discuss next steps.</Text>
+            <Card
+              p={4}
+              bg={theme.colors.slate['50']}
+              borderWidth="1px"
+              borderColor={theme.colors.slate['200']}
+              borderRadius="lg"
+              variant="outline"
+            >
+              <VStack spacing={4}>
+                <Box w="100%">
+                  <Text fontSize="xs" color={theme.colors.slate['700']}>
+                    PATIENT
+                  </Text>
+                  <Text fontSize="sm" fontWeight="medium">
+                    {order?.patient && order.patient.name.full}
+                  </Text>
+                </Box>
+                <VStack w="100%" align="left">
+                  <Text fontSize="xs" color={theme.colors.slate['700']}>
+                    PRESCRIPTION
+                  </Text>
+                  {fills?.length > 0 &&
+                    fills.map((fill) => (
+                      <Box w="100%" key={fill.treatment.name}>
+                        <Text fontSize="sm" fontWeight="medium">
+                          {fill?.treatment?.name}
+                        </Text>
+                        <Text fontSize="sm" color={theme.colors.slate['500']}>
+                          {formatRxString({
+                            dispenseQuantity: fill?.prescription?.dispenseQuantity,
+                            dispenseUnit: fill?.prescription?.dispenseUnit,
+                            fillsAllowed: fill?.prescription?.fillsAllowed,
+                            instructions: fill?.prescription?.instructions
+                          })}
+                        </Text>
+                      </Box>
+                    ))}
+                </VStack>
+              </VStack>
+            </Card>
+          </>
+        }
+        prependContext={
+          !order
+            ? ''
+            : `
+Order:
+  ID: ${order.id}
+
+----
+Patient:
+  ID: ${order?.patient?.id} 
+  Name: ${order?.patient?.name?.full}
+
+----
+Prescriptions:
+${fills.map(
+  (fill) => `
+  Name: ${fill.treatment.name}
+  Info: ${formatRxString({
+    dispenseQuantity: fill?.prescription?.dispenseQuantity,
+    dispenseUnit: fill?.prescription?.dispenseUnit,
+    fillsAllowed: fill?.prescription?.fillsAllowed,
+    instructions: fill?.prescription?.instructions
+  })}
+`
+)}
+
+---- 
+Description: 
+        `
+        }
+      />
```

Comment:
Can we put this in its own file / component?

Reply:
You mean specifically the template literal? I can move it out